samples/adminconfig/infrastructure.json

```diff
@@ -1,45 +1,47 @@
 {
+    "definitions": [
+        {
+            "attribute": "storage-cost",
```

Comment:
Do we need to map to the particular attribute type?  If for example, I have a price that is in us dollars with a scale of 0..100, couldn't I in theory use it with different attribute types?

Reply:
Moved the attribute out. The connection to the metrics is done using metrics name.